metadata.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "hunner-wordpress",
+  "version": "1.0.0",
+  "author": "Hunter Haugen",
+  "summary": "Puppet module to set up an instance of wordpress; and optionally a mysql db/user.",
+  "license": "Apache-2.0",
+  "source": "https://github.com/hunner/puppet-wordpress",
+  "dependencies": [
+    {"name":"puppetlabs/concat",    "version_requirement":">= 1.0.0"},
+    {"name":"puppetlabs/mysql",     "version_requirement":">= 2.1.0"},
+    {"name":"puppetlabs/stdlib",    "version_requirement":">= 2.3.1"}
+  ],
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "CentOS",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "OracleLinux",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Scientific",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Debian",
+      "operatingsystemrelease": [
+        "8"
+      ]
+    },
+    {
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "16.04"
+      ]
+    }
+  ],
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 4.7.0 < 6.0.0"
+    }
+  ],
+  "pdk-version": "1.4.1",
+  "template-url": "file:///opt/puppetlabs/pdk/share/cache/pdk-templates.git",
```

Comment:
Can you point this at the github url instead of the /opt/puppetlabs one?

Reply:
That's odd. I wonder why that's not the default.

Reply:
We don't default to the url because we want the default templates to be mapped to the pdk cached templates to support non-networked or airgapped environments.

Reply:
@bmjen I don't care about supporting airgapped environments doing development on my module. I care only about networked environments, basically.

Is there a reason to not have a github template url in public modules? When users are doing development I'd rather have the most up-to-date templates ready to go and also be portable between git installs and windows installs as well. For private/internal modules with defined operating environments I could see `file://` urls, but not publicly-developed modules.

Is there a case when github urls would blow up in our faces?

Reply:
In fact, I thought it might be nice for the forge rating system to give a warning when a non-portable template-url was given...

Reply:
No reason, supported modules **should** have the github url in the template-url field. You'd just have to do the `pdk convert` for the module with the `--template-url` option. My comment was merely stating why we do the `file:///` by default when using the all-in-one installer.

Reply:
Oh I see, thanks.